src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,118 @@
+Assignment #5: Launch Checklist Form
+====================================
+
+Using our knowledge of forms, the DOM, and HTTP, our favorite space shuttle program has asked us to create a quick launch checklist.
+We have four fields that need to be filled out with vital information: the pilot's name, the co-pilot's name, the fuel levels, and the weight of the cargo.
+
+Our pilot, Chris, and the co-pilot, Blake, have been hard at work securing the cargo and filling the shuttle tank. All we need to do is use validation to ensure that we have all of the info for the space shuttle program and make sure no one prematurely launches the shuttle.
+
+Requirements
+------------
+
+Create a Launch Checklist Form for astronauts to fill out in preparation for launch.
+The form should do the following:
+
+1. Validate the user responses with ``preventDefault()`` to ensure the following:
+
+   a. The user entered something for every field.
+   b. The user entered text for names and numbers for fuel and cargo levels.
+
+2. With validation, update a list of what is currently ready or not ready for the shuttle launch.
+3. Indicate what is good or bad about the shuttle and whether it is ready for launch by using the DOM to update the CSS.
+4. Fetch some planetary JSON to update the mission destination with vital facts and figures about where the shuttle is headed. 
+
+Setting Up Your Project Repository
+----------------------------------
+
+Fork the repository with the `starter code <https://github.com/LaunchCodeEducation/Launch-Checklist-Form/>`_ to your personal Github profile and clone the repository to the directory where you are keeping your assignments for the class.
+
+.. warning::
+
+   When updating styles to indicate whether the shuttle is ready for launch, do not modify ``styles.css``!
```

Comment:
Caps emphasis: "...do NOT modify ``styles.css``!"

Reply:
Done! Thanks!